src/Core/Grid/Presenter/GridPresenter.php

```diff
@@ -133,29 +134,23 @@ private function presentBulkActions(GridInterface $grid)
      *
      * @return array
      */
-    private function presentGridActions(GridInterface $grid)
+    private function presentPanelAction(GridInterface $grid)
     {
-        $gridActionsArray = [];
+        $panelActions = [];
 
-        /** @var GridActionInterface $gridAction */
-        foreach ($grid->getDefinition()->getGridActions() as $gridAction) {
+        /** @var PanelActionInterface $panelAction */
+        foreach ($grid->getDefinition()->getPanelActions() as $panelAction) {
```

Comment:
in fact everytime you want to loop into a collection, this probably means you want to add a new public method into that collection class :+1: 

This is the biggest interest of having "typed" lists